runtests.sh

```diff
@@ -27,6 +27,7 @@ PYTHONPATH=${HYPERNODE_VAGRANT_RUNNER_DIR}/tools/hypernode-vagrant-runner \
     --project-path="$PROJECT_DIRECTORY" \
     --command-to-run="$TEST_COMMAND" \
     --user="$TEST_USER" \
+    --xenial \
```

Comment:
should we not default this in the hypernode-vagrant-runner?

Reply:
yes but then we'd break anyone depending on it which is something to consider

Reply:
that's a valid point yeah. external parties maybe using it